ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/LedgerApiBenchTool.scala

```diff
@@ -37,8 +37,6 @@ object LedgerApiBenchTool {
               logger.info(s"Benchmark finished successfully.")
             case Left(error) =>
               logger.info(s"Benchmark failed: $error")
-              // Exit with error status code to facilitate scripting
-              sys.exit(1)
```

Comment:
How will the SLOs be used then? Will the caller have to parse the log output to see if the benchmark failed?